hera_qm/tests/test_firstcal_metrics.py

```diff
@@ -188,20 +196,6 @@ def test_firstcal_metrics_run(self):
         self.assertTrue(os.path.exists(dest_file))
         os.remove(dest_file)
 
-        # test w/ rotant json
```

Comment:
In the future let's try to keep branches/PRs limited to a single idea, and make a separate PR for handling other stuff that's found in the process.